scripts/manifest

```diff
@@ -7,7 +7,7 @@ fi
 
 set -e -x
 
-export REPO="rancher/k3s-upgrade"
+export REPO="k3s-io/k3s-upgrade"
```

Comment:
this is meant to be the github repo and not the docker hub repo? because the docker hub repo is still rancher

Reply:
no, looking below it is definitely the docker hub repo